core/src/test/scala/integration/kafka/api/SaslPlainSslEndToEndAuthorizationTest.scala

```diff
@@ -147,10 +157,11 @@ class SaslPlainSslEndToEndAuthorizationTest extends SaslEndToEndAuthorizationTes
 
   /**
    * Checks that secure paths created by broker and acl paths created by AclCommand
-   * have expected ACLs.
+   * have expected ACLs. This only applies for zookeeper
```

Comment:
why does this only work in zk mode?

Reply:
There isn't a zookeeper so while the test would pass, it wouldn't actually test anything.